gslib/utils/cat_helper.py

```diff
@@ -137,6 +137,10 @@ def CatUrlStrings(self,
               printed_one = True
             cat_object = blr.root_object
             storage_url = StorageUrlFromString(blr.url_string)
```

Comment:
This line can be moved below the new `if` block because it doesn't matter if the `if` block triggers a `return`.